### PR TITLE
fix flaky test on standalone

### PR DESCRIPTION
**Pull request by tpereirasplunk**

### CODE REVIEW

File: test/appframework_aws/s1/appframework_aws_test.go [hunk 1]
```diff
@@ -1723,19 +1724,9 @@ var _ = Describe("s1appfw test", func() {
...
 			appVersion = "V1"
-			appFileList = testenv.GetAppFileList(appListV1)
-			standalonePod := []string{fmt.Sprintf(testenv.StandalonePod, deployment.GetName(), 0)}
-			standaloneAppSourceInfo := testenv.AppSourceInfo{CrKind: standalone.Kind, CrName: standalone.Name, CrAppSourceName: appSourceName, CrPod: standalonePod, CrAppVersion: appVersion, CrAppScope: enterpriseApi.ScopeLocal, CrAppList: appListV1, CrAppFileList: appFileList}
-			allAppSourceInfo := []testenv.AppSourceInfo{standaloneAppSourceInfo}
-			testenv.AppFrameWorkVerifications(ctx, deployment, testcaseEnvInst, allAppSourceInfo, splunkPodAge, "")
+			testenv.VerifyAppInstalled(ctx, deployment, testcaseEnvInst, testcaseEnvInst.GetName(), []string{fmt.Sprintf(testenv.StandalonePod, deployment.GetName(), 0)}, appListV1, false, "enabled", false, false)
 
 			// Check for changes in App phase to determine if next poll has been triggered
 			appFileList = testenv.GetAppFileList(appListV2)
```
Comment (jambrosiano) [inline test/appframework_aws/s1/appframework_aws_test.go:1732]: we got this 3 times on lines 1721,1732,1743 but once is enough in this case, isn't it?

File: test/appframework_aws/s1/appframework_aws_test.go [hunk 2]
```diff
@@ -1683,6 +1683,7 @@ var _ = Describe("s1appfw test", func() {
 			// ################## SETUP FOR STANDALONE ####################
 			// Download all test apps from S3
 			appVersion := "V1"
+			appListV1 := []string{appListV1[0]}
```
Comment (jambrosiano) [inline test/appframework_aws/s1/appframework_aws_test.go:1686]: So, for V1 we take 1 app only, but for V2 later on we take the normal appListV2?
Is that what we want to do?

**Comment by jambrosiano** [comment]
We would need this in appframework_az folder also btw

File: test/appframework_aws/s1/appframework_aws_test.go (hunk 2 above)
Comment (tpereirasplunk) [reply test/appframework_aws/s1/appframework_aws_test.go:1686]: The test case involves updating the app as soon as we download it. If there are too many apps by the time we download the updated apps, the apps installed get updated and we are unable to check that previous install took place.

This added flakiness to the test. Changing the number of apps in this test case to counteract this issue

**Review by jambrosiano — APPROVED** [review]

**Review by gaurav-splunk — APPROVED** [review]

**Pull request merged**